src/constants.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 export class Constants {
-    public static DesignerVersion = "1.40121.1.503775.190124-1744";
+    public static DesignerVersion = "1.40128.1.31206347.190129-1631";
```

Comment:
Pending Test sign off of this week's BPMUX release.

Reply:
There was one more fix which made it into this week's release today in so I'll update the version after tonight's test pass.